extensions/windows/cfapi/Vfs/Vfs.vcxproj

```diff
@@ -189,11 +189,8 @@ copy /Y "$(TargetDir)$(TargetName).pdb" "F:\Projects\build-kdrive-Desktop_Qt_6_2
     </Link>
     <Manifest />
     <PostBuildEvent>
-      <Command>copy /Y "$(ProjectDir)..\Common\debug.h" "F:\Projects\kdrive\src\server\vfs\win\."
-copy /Y "$(ProjectDir)$(TargetName).h" "F:\Projects\kdrive\src\server\vfs\win\."
```

Comment:
Should we adapt the build instructions for Windows? If I am correct, users who want to install `desktop-kDrive` are invited to edit the Build Events of the `Vfs` project in the documentation, i.e., to replace  `F:\Projects` with the appropriate folder.

Reply:
The Post-Build instructions need to be updated to remove that hardcoded`F:\Projects` path in Debug, however this change is only removing all the Post-Build instructions for **Release** builds, as the extension file management is done by my powershell script